src/stores/grid_renderer_store.ts

```diff
@@ -713,7 +722,7 @@ export class GridRenderer {
     return box;
   }
 
-  private getGridBoxes(): Box[] {
+  private getGridBoxes(renderingContext: GridRenderingContext): Box[] {
```

Comment:
I'm pretty sure someone once told me that we wanted to keep the creation of grid boxes independant from the `ctx`. Was it @rrahir ?

We could easily have the indent be `fontsize * randomConstant`, rather to bother having to measure space characters with the context